views/index.html

```diff
@@ -1,89 +1,93 @@
-        <button class="calculator-orange-button white-operator">/</button>
-      </div>
-    </div>
+  <div id="calculator-numbers">
 
-    <div class="calculator-row">
-      <div class="calculator-button">
-        <button>7</button>
-      </div>
-      <div class="calculator-button">
-        <button>8</button>
+      <div class="calclulator-row1">
+        <div class="calculator-button">
+          <button class="calculator-ac">AC</button>
+        </div>
+        <div class="calculator-button">
+          <button class="calculator-positive-negative">+/-</button>
+        </div>
+        <div class="calculator-button">
+          <button class="percent">%</button>
+        </div>
+        <div class="calculator-button">
+          <button class="calculator-orange-button white-operator divide">/</button>
+        </div>
       </div>
-      <div class="calculator-button">
-        <button>9</button>
-      </div>
-      <div class="calculator-button">
-        <button class="calculator-orange-button white-operator">x</button>
-      </div>
-    </div>
 
-    <div class="calculator-row3">
-      <div class="calculator-button">
-        <button>4</button>
-      </div>
-      <div class="calculator-button">
-        <button>5</button>
-      </div>
-      <div class="calculator-button">
-        <button>6</button>
+      <div class="calculator-row2">
+        <div class="calculator-button">
+          <button id="button7" onclick='eventLogger'>7</button>
+        </div>
+        <div class="calculator-button">
+          <button id="button8" onclick='eventLogger'>8</button>
+        </div>
+        <div class="calculator-button">
+          <button id="button9" onclick='eventLogger'>9</button>
+        </div>
+        <div class="calculator-button">
+          <button class="calculator-orange-button white-operator multiply">x</button>
+        </div>
       </div>
-      <div class="calculator-button">
-        <button class="calculator-orange-button white-operator">-</button>
-      </div>
-    </div>
 
-    <div class="calculator-row">
-      <div class="calculator-button">
-        <button>1</button>
-      </div>
-      <div class="calculator-button">
-        <button>2</button>
-      </div>
-      <div class="calculator-button">
-        <button>3</button>
-      </div>
-      <div class="calculator-button">
-        <button class="calculator-orange-button white-operator">+</button>
+      <div class="calculator-row3">
+        <div class="calculator-button">
+          <button id="button4" onclick='eventLogger'>4</button>
+        </div>
+        <div class="calculator-button">
+          <button id="button5" onclick='eventLogger'>5</button>
+        </div>
+        <div class="calculator-button">
+          <button id="button6" onclick='eventLogger'>6</button>
+        </div>
+        <div class="calculator-button">
+          <button class="calculator-orange-button white-operator subtract">-</button>
```

Comment:
It's just the color of the text on the button. The operators are the mathematic operators